### PR TITLE
Feat/pci uuids

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7096, MANAGER-7088, MANAGER-7086
| License          | BSD 3-Clause

## Related

- [x] TRANS-36019

### CODE REVIEW

**Review by frenautvh — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/data-processing/data-processing.controller.js [hunk 1]
```diff
@@ -51,8 +54,9 @@ export default class {
    * @return {*|Promise<any>}
    */
   getJobs({ offset, pageSize, sort, criteria }) {
+    console.log(criteria);
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/data-processing/data-processing.controller.js:57]: No :D

Comment (marie-j) [reply packages/manager/modules/pci/src/projects/project/data-processing/data-processing.controller.js:57]: Forgot this one 🤭

File: packages/manager/modules/pci/src/projects/project/analytics-data-platform/analytics-data-platform.routing.js [hunk 2]
```diff
@@ -15,8 +15,14 @@ export default /* @ngInject */ ($stateProvider) => {
...
         ),
-
+    params: {
+      id: {
+        dynamic: true,
+        type: 'string',
+      },
+    },
     resolve: {
+      clusterId: /* n@gInject */ ($transition$) => $transition$.params().id,
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/analytics-data-platform/analytics-data-platform.routing.js:25]:
```suggestion
      clusterId: /* @ngInject */ ($transition$) => $transition$.params().id,
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report translations-checker#17928.0 ✘  
* Stage 1
  * Check ✘

**Review by antleblanc — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**